README.md

```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Commands
+| Command                                              | Description                                                                                          |
+| ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [crio(8)](/docs/crio.8.md)                 | Enable OCI Kubernetes Container Runtime daemon |
```

Comment:
The first line of each of these pages are not consistent and should be.  Some are bold, some have dates, some have authors, others do not.  Also some have their number a "1" or "8" and it should be (1) or (8).  The changes here in this particular page are fine, so I'm good with getting this submitted and creating a follow up/issue to make these called pages consistent.